### DISCUSSION TITLE
How to annotate broad search queries?

### DISCUSSION THREAD

**Discussion by bernikr** [sebastian-hofstaetter/teaching#52] [Advanced IR: Exercise Q&A] [answered]
I started with the annotation work and I am a bit confused on how to annotate most queries. In all the examples the queries are always a specific search for something that needs to be answered. But in reality most of the queries (i have done around 15 now) are just broad topics such as "ukraine foreign affairs" or "turkey news". If we are now presented with a news article related to this topic should this be classified as "topic" or "partial" (or even "perfect"). Also it is almost impossible to highlight specific passages for such broad queries.

**Comment by DezignDezire** [comment]
Agreed, I predict a high correlation between the length of the query and the rating of the sample. You should probably normalize the results during evaluation accordingly.

**Comment by LaurenzV** [comment]
I agree, I have annotated 50+ queries so far, and in most of the cases I was very unsure as to what would actually be an appropriate query, since depending on how you see it, anything would go... Like for example, one query was  "london real estate" but the article was only about the real estate market in Jordan. In this case, should I cross "Topic" because the topic "real estate" still fits, or should I choose "Wrong" (what I ended up doing) since it simply isn't about the real estate market in London? In other cases, the articles did talk about the topic in the query, but they were so long that I couldn't really decide between "Partial" and "Perfect" (or even just "Topic"). It would have been helpful to get more detailed instructions for such cases.

**Comment by sophiaalthammer** [comment, marked as answer]
Hi everyone, 
you are right, we should have clarified this more detailed in the beginning. So as you already say, the queries are topical related to the article or partially relevant to the document, therefore I would also annotate them as Topic or Partial, depending on how relevant the query is to the document and if another query could be more relevant to the document. I would see it that a Perfect relevance is given if the document exactly answers the information need of the query.

I hope this clarification helps, if not let us know!

**Reply by tkunanuntakij** [reply]
What about if a query is board but the result is more specific. For example, if the query is just about dog but the text talk about a specific breed of dogs or like in the `LauranzV` example? Should this be marked as `Perfect` or `Partial`.

**Reply by sophiaalthammer** [reply]
This should be annotated as Partial.
